### PR TITLE
Add introduction to distance and angles

### LINKED ISSUE #15: Add introduction to distance and angles
Related to https://github.com/seismo-learn/software/issues/69.

### PR DESCRIPTION
**Pull request by core-man**
Closes #15.

### CODE REVIEW

**Comment by seisman** [comment]
This is not what I expect. I thought they should be in a subsection, similar to what you did here (https://migg-ntu.github.io/SeisTomo_Tutorials/seismology/seismic-station/index.html#distance-and-angles).

**Comment by core-man** [comment]
> This is not what I expect. I thought they should be in a subsection, similar to what you did here (https://migg-ntu.github.io/SeisTomo_Tutorials/seismology/seismic-station/index.html#distance-and-angles).

Please check the new structure.

File: source/seismograph/index.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+地震仪
```
Comment (seisman) [inline source/seismograph/index.rst:1]: Distance, azimuth and back-azimuth are not attributes of a seismograph, but I also have a better idea where to put them.

Comment (core-man) [reply source/seismograph/index.rst:1]: Shall we add an appendix in the seismology chapter? But the name should not be appendix cause we already have an appendix for the whole document. How about 名词解释？

Comment (seisman) [reply source/seismograph/index.rst:1]: We can still put it here, and move it to a better place when we finish the source, and station part.

Comment (core-man) [reply source/seismograph/index.rst:1]: Okay.

File: source/seismograph/index.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+======
+
+
+距离和角度
+----------
+
+震中距
+++++++
+
+地震、球面上任意一点和球心所在的球面圆形称为大圆（*great circle*）。
```
Comment (seisman) [inline source/seismograph/index.rst:11]: This sentence doesn't read well.

File: source/seismograph/index.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+大圆由从震中到这一点的小圆弧（*minor arc*）和大圆弧（*major arc*）
+两部分组成。
+
+震中距指震中到地面上任意一点的球面距离，即小圆弧或大圆弧的长度，单位一般
+为度或公里等。除非显示申明，一般默认是小圆弧长度。
```
Comment (seisman) [inline source/seismograph/index.rst:16]: We should mention that 1°=111.2 km, and km is commonly used for local earthquakes, and degree is used for tele earthquakes.

File: source/seismograph/index.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+距离和角度
+----------
+
+震中距
+++++++
+
+地震、球面上任意一点和球心所在的球面圆形称为大圆（*great circle*）。
+大圆由从震中到这一点的小圆弧（*minor arc*）和大圆弧（*major arc*）
```
Comment (seisman) [inline source/seismograph/index.rst:12]:
```suggestion
大圆由从震中到这一点的小圆弧（*minor arc*）和大圆弧（*major arc*）
```
I thought they're 优弧 and 劣弧。

File: source/seismograph/index.rst [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+两部分组成。
+
+震中距指震中到地面上任意一点的球面距离，即小圆弧或大圆弧的长度，单位一般
+为度或公里等。除非显示申明，一般默认是小圆弧长度。
+
+方位角和反方位角
+++++++++++++++++
+
+方位角是震中到台站的连线与地理北向的夹角，而反方位角指台站到震中的连线与地理北向的夹角。
```
Comment (seisman) [inline source/seismograph/index.rst:21]: It would be better to emphasize that az is not equal to baz+180 or similar.

**Review by seisman — APPROVED** [review]

**Pull request merged**